Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:8-jdk-alpine
+
+# Set working directory
+WORKDIR /app
+
+# Copy current directory contents into container
+COPY . /app
+
+# port to expose
+EXPOSE 80
+
+RUN dos2unix gradlew
+RUN ./gradlew build
```

Comment:
no, we are not building the jar in this docker file, the build should be done by circle ci otherwise every time we lauch the app we have to build it!